packages/sui-studio-utils/README.md

```diff
@@ -44,6 +44,25 @@ domain.get('current_user_use_case').execute().then((products) => {
 })
 ```
 
+#### more than one use case at a time
+
+```js
+import { DomainBuilder } from '@s-ui/studio-tools'
+import myDomain from 'domain'
+
+const domain = DomainBuilder.extend({ myDomain })
+  .mockUseCases([
```

Comment:
What about calling multiple time to the function `for()` instead of having 2 different methods of adding mocks?
Could it be something like
`
DomainBuilder.extend({ myDomain }).for({useCase: 'get_products_use_case'}).respondWith(response).and().for({useCase: 'get_user_use_case'}).respondWith(userResponse).build()
`